### ISSUE TITLE
elastic-agent swallows stderr and panics

### ISSUE THREAD

**Issue by mtojek** [elastic/beats#30062]
Hi Team,

this [issue](https://github.com/elastic/integrations/issues/2602#issuecomment-1023268069) proved that we need a more user-friendly method to collect all panics. Otherwise we won't be able to grab anything from customers.

In the mentioned case I used `strace` to listen what `filebeat` is sending to `/dev/null`, but we need a more trival method.

**Comment by jlind23** [comment]
@ruflin @ph From your opinion would it be something easy to put in place?

**Comment by ph** [comment]
I don't think it would be much effort and this would align with v2 changes for logging. IE: We should be able to stream `stderr` from a subprocess into the Elastic Agent Appender.

**Comment by ruflin** [comment]
I like the idea of starting to feed this into the elastic-agent logs. This will help us to already gain some experience around how it will work in v2 as @ph pointed out.
